src/Sylius/Bundle/ShopBundle/Resources/views/Product/Show/_associations.html.twig

```diff
@@ -1,2 +1,9 @@
-<h4 class="ui horizontal section divider header">{{ 'sylius.ui.you_may_also_like'|trans }}</h4>
-{{ render(url('sylius_shop_partial_product_index_latest', {'count': 4, 'template': '@SyliusShop/Product/_simpleList.html.twig'})) }}
+{% for association in product.associations %}
+    <div>
+        <h4 class="ui horizontal section divider header">{{ association.type.name }}</h4>
+        {{ render(url('sylius_shop_partial_associated_products', {'id': association.id, 'template': '@SyliusShop/Product/_associations.html.twig'})) }}
```

Comment:
I'd also go for a partial for this kind of components of a product page, because I'd use HTTP cache to avoid making the product page heavy to generate.

this is something we do for example with menus that are generated dinamically at bestvalue.eu - we rely on a subrequest that is cached for 24h.